backtrace-library/src/main/java/backtraceio/library/models/json/BacktraceReport.java

```diff
@@ -185,7 +185,11 @@ public BacktraceReport(
     }
 
     public BacktraceData toBacktraceData(Context context, Map<String, Object> clientAttributes) {
-        return new BacktraceData(context, this, clientAttributes);
+        return new BacktraceData(context, this, clientAttributes, false);
```

Comment:
here you should call below method `toBacktraceData(context, this, clientAttributes, false) {`